device/mellanox/x86_64-mlnx_msn2700-r0/psu_sensors.conf

```diff
@@ -0,0 +1,427 @@
+{
+    "platform": {
+        "MSN2010": {
```

Comment:
In the scripts will skip these two platforms, seems to me no need to add these two. it will cause misunderstanding?

Reply:
removed